src/main/java/billennium/tests/entity/Answer.java

```diff
@@ -10,7 +10,7 @@
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
 
-@Data
+@Data //TODO  brak konstruktora dla wszystich paramsow. Brak bezparametrowego ktory jest wymagany do tworzenia encji
```

Comment:
ogólnie zamysł miałem by nie ustawiać atrybutów encji przez konstruktor. To tak naprawdę zależy co wybierzemy, wydaje mi się zdania o to czy encja powinna być mutable jest podzielona.

Reply:
1Ważna rzecz: aby encja była encja w javie musi miec konstruktor bezparametrowy.